File: 1768.cpp
#include <string>
using namespace std;

class Solution {
public:
  string mergeAlternately(string word1, string word2) {
    std::string newWord;
    for (int i = 0; i <= (word1.length() + word2.length()); i++) {
      if (i < word1.length()) {
        newWord += word1[i];
      }
      if (i < word2.length()) {
        newWord += word2[i];
      }
    }
    return newWord;
  }
};

File: 1768.ts
function mergeAlternately(word1: string, word2: string): string {
  let size = word1.length + word2.length;
  let new_word = "";
  for (let i = 0; i < size; i++) {
    if (i < word1.length) {
      new_word += word1[i];
    }
    if (i < word2.length) {
      new_word += word2[i];
    }
  }
  return new_word;
}
